Build fails on mingw

In file included from C:/Users/kreij/scoop/apps/msys2/2022-01-28/home/kreij/stable-diffusion.cpp/util.cpp:1:
C:/Users/kreij/scoop/apps/msys2/2022-01-28/home/kreij/stable-diffusion.cpp/util.h:20:1: error: 'int32_t' does not name a type
   20 | int32_t get_num_physical_cores();
      | ^~~~~~~
C:/Users/kreij/scoop/apps/msys2/2022-01-28/home/kreij/stable-diffusion.cpp/util.h:5:1: note: 'int32_t' is defined in header '<cstdint>'; did you forget to '#include <cstdint>'?
    4 | #include <string>
  +++ |+#include <cstdint>
    5 |
make[2]: *** [CMakeFiles/stable-diffusion.dir/build.make:104: CMakeFiles/stable-diffusion.dir/util.cpp.obj] Error 1
make[1]: *** [CMakeFiles/Makefile2:167: CMakeFiles/stable-diffusion.dir/all] Error 2
make: *** [Makefile:136: all] Error 2

---
It gets fixed if you import <cstdint> before using int32_t

--- a/util.h
+++ b/util.h
@@ -2,6 +2,7 @@
 #define __UTIL_H__
 
 #include <string>
+#include <cstdint>
 
 bool ends_with(const std::string& str, const std::string& ending);
 bool starts_with(const std::string& str, const std::string& start);

